src/test/resources/allfields/output/input-ruimtereis01/deposit.properties

```diff
@@ -13,3 +13,4 @@ springfield.user=janvanmansum
 springfield.playmode=menu
 state.description=Deposit is valid and ready for post-submission processing
 state.label=SUBMITTED
+identifier.doi.registered=no
```

Comment:
Could you add newlines at the end of these files. Otherwise we get weird diffs next time...

Also turn on the setting in IntelliJ to add newlines to the end of files on save.

Reply:
Done